README.md

```diff
@@ -40,7 +40,6 @@ Drivers facilitate interactions with external resources and services:
 - 🔢 **Embedding Drivers** generate vector embeddings from textual inputs.
 - 💾 **Vector Store Drivers** manage the storage and retrieval of embeddings.
 - 🎨 **Image Generation Drivers** create images from text descriptions.
-- 🔎 **Image Query Drivers** query images from text queries.
```

Comment:
Up to you on this, but may be good to add a line in the prompt drivers that they can now handle multi-modal queries or something so it doesn't seem like we don't support it at all.

Reply:
Maybe this list could focus on functionality (like verbs), then mention which driver gives it to you 🤷 .